src/react-queries/hooks/useQuoteEstimatedFeesQuery.ts

```diff
@@ -27,6 +39,14 @@ type UseQuoteEstimatedFeesProps = {
       repaymentAccountId: AccountId
       repaymentAsset: Asset | null
     }
+  | {
+      confirmedQuote: LpConfirmedDepositQuote | null
+      // Technically not a collateral, but this avoids weird branching, ternaries or ?? check sfor now
```

Comment:
If we really don't like that smell of this, this can be cleaned up and make `const accountId = repaymentAccountId ?? collateralAccountId` an IIFE instead

Reply:
<img src="https://media1.giphy.com/media/NTur7XlVDUdqM/giphy.gif"/>